guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -451,8 +452,8 @@ Use this option if you have one of the following errors:
 . In the {Project} web UI, navigate to *Content* > *Products*.
 . Select the product containing the corrupted repository.
 . Select the name of a repository you want to synchronize.
-. From the *Select Action* menu, select *Advanced Sync*.
-. Select the option and click *Sync*.
+. To verify the checksum, click *Verify Content Checksum* from the *Action* menu.
+. To perform optimised sync and complete sync, select *Advanced Sync* from the *Select Action* menu. And then, select the option and click *Sync*.
```

Comment:
We use American English in the docs so ized rather than ised. 
We don't start paragraphs with And. 
What option? we should tell them what to select.